### PR TITLE
convert to standard and clean up makefile

### PR DESCRIPTION
**Pull request by stevenmiller888**
cc @segmentio/gateway 

This converts the codebase to [standard](https://github.com/feross/standard), removes eslint, and cleans up the makefile a bit.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,44 +19,27 @@ endif
...
-
-# Install dependencies.
-install: node_modules
+	npm install
+	touch $@
 
-# Remove temporary files and build artifacts.
 clean:
 	rm -rf *.log coverage
-.PHONY: clean
```
Comment (Rowno) [inline Makefile]: I prefer separate `.PHONY` statements so that you don't forget to add them but ¯\_(ツ)_/¯

File: karma.conf.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ module.exports = function(config) {
 
     frameworks: ['browserify', 'mocha'],
 
-    reporters: ['spec'/* , 'coverage' */],
+    reporters: ['spec'],
```
Comment (Rowno) [inline karma.conf.js]: Shouldn't coverage be enabled? 😛

File: lib/analytics.js [hunk 3]
```diff
@@ -580,16 +579,16 @@ Analytics.prototype._callback = function(fn) {
...
+Analytics.prototype._invoke = function (method, facade) {
+  this.emit('invoke', facade)
 
-  each(function(integration, name) {
-    if (!facade.enabled(name)) return;
-    integration.invoke.call(integration, method, facade);
-  }, this._integrations);
+  each(function (integration, name) {
+    if (!facade.enabled(name)) return
+    integration.invoke.call(integration, method, facade) // eslint-disable-line no-useless-call
```
Comment (Rowno) [inline lib/analytics.js]: Couldn't this just be `integration.invoke(method, facade)`?

File: karma.conf.js (hunk 2 above)
Comment (stevenmiller888) [inline karma.conf.js]: It was already disabled. Not changing that :)

File: lib/analytics.js (hunk 3 above)
Comment (stevenmiller888) [inline lib/analytics.js]: Yes, but it causes test to fail

Comment (Rowno) [inline lib/analytics.js]: That's really weird. 😕

Comment (stephenmathieson) [inline lib/analytics.js]: Can we add a comment explaining that? This is super weird/sounds like a bug.

**Comment by f2prateek** [comment]
What's the status of this? No go?

**Comment by f2prateek** [comment]
Tracking in JIRA https://segment.atlassian.net/browse/LIB-144. One thing I'm unsure around this is the compatibility (for integrations and customers particularly).

**Pull request closed without merging**